motrackers/detectors/yolo.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, weights_path, configfile_path, labels_path, confidence_thresh
         object_names = load_labelsjson(labels_path)
 
         layer_names = self.net.getLayerNames()
-        if cv2.__version__ == '4.6.0':
+        if cv.__version__ == '4.6.0':
```

Comment:
Could we also make this check for versions greater than or equal to `4.6.0`?
```suggestion
        if cv.__version__ >= '4.6.0':
```